src/main/scala/models/app/MicrosoftSynapseLinkStreamContext.scala

```diff
@@ -129,14 +129,27 @@ case class MicrosoftSynapseLinkStreamContext(spec: StreamSpec) extends StreamCon
 
   val stagingTableNamePrefix: String = spec.stagingDataSettings.tableNamePrefix
   val stagingCatalog: String = s"${spec.stagingDataSettings.catalog.catalogName}.${spec.stagingDataSettings.catalog.schemaName}"
-  
+
   val partitionExpressions: Array[String] = spec.tablePropertiesSettings.partitionExpressions
   val format: TableFormat = TableFormat.valueOf(spec.tablePropertiesSettings.format)
   val sortedBy: Array[String] = spec.tablePropertiesSettings.sortedBy
   val parquetBloomFilterColumns: Array[String] = spec.tablePropertiesSettings.parquetBloomFilterColumns
 
 given Conversion[StreamSpec, CdmTableSettings] with
   def apply(spec: StreamSpec): CdmTableSettings = CdmTableSettings(spec.sourceSettings.name.toLowerCase, spec.sourceSettings.baseLocation)
+  
+given Conversion[TablePropertiesSettings, Map[String, String]] with
```

Comment:
Not sure if we need this conversion here, we can make `TableManager` to consume `TableProperriesSetings` and do the explicit conversion there.

`Map[String, String]` is too generic type to me for implicit conversion

Reply:
fixed :)